.github/workflows/autofix.yml

```diff
@@ -15,7 +15,7 @@ jobs:
 
       - uses: pnpm/action-setup@v2
         with:
-          version: 8
+          version: 9.7.1
```

Comment:
Should match https://github.com/anti-work/shortest/blob/08a763477efe26b6b7fcd2d4523030edbd32d230/package.json#L4